linkis-computation-governance/linkis-entrance/src/main/java/com/webank/wedatasphere/linkis/entrance/restful/EntranceRestfulApi.java

```diff
@@ -257,18 +257,18 @@ public Response log(@Context HttpServletRequest req, @PathParam("id") String id)
                     retLog = sb.toString();
                 }
             } catch (IllegalStateException e) {
-                logger.error("为 {} 获取日志失败 原因：{}", job.get().getId(), e.getMessage());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId(), e.getMessage());
                 message = Message.ok();
                 message.setMethod("/api/entrance/" + id + "/log");
                 message.data("log", "").data("execID", id).data("fromLine", retFromLine + fromLine);
             } catch (final IllegalArgumentException e) {
-                logger.error("为 {} 获取日志失败", job.get().getId());
+                logger.error("Failed to get log information for :{}(为 {} 获取日志失败)", job.get().getId(), job.get().getId());
                 message = Message.ok();
```

Comment:
The original code here has BUGs,  the return message should use Message.error. Please help optimize it.